### DISCUSSION TITLE
How to run `kubeone apply` without interactive prompt `yes`?

### DISCUSSION THREAD

**Discussion by blindpirate** [kubermatic/kubeone#2403] [Q&A] [answered]
```
Run with --verbose flag for more information.
        + initialize control plane node "hetzner-k8s-control-plane-1" (192.168.0.3) using 1.24.6
        + ensure machinedeployment "hetzner-k8s-pool1" with 10 replica(s) exists

Do you want to proceed (yes/no): yes
```

I want to run this in an automation script without human interaction. `terraform apply -auto-approve` can be used to avoid manually inputting `yes`, but I don't find equivalent parameter for `kubeone apply`, can anyone help me?

**Comment by kron4eg** [comment, marked as answer]
> `terraform apply -auto-approve` can be used to avoid manually inputting `yes`, but I don't find equivalent parameter for `kubeone apply`, can anyone help me?

```
$ kubeone apply --help | grep auto
  -y, --auto-approve                  auto approve plan
```
